### ISSUE TITLE
Error on template.content.cloneNode(true)

### ISSUE THREAD

**Issue by timonson** [b-fuze/deno-dom#145]
Hello,

I would like to clone the HTMLTemplateElement contents and get the following error although it is valid JavaScript.

```ts
      const template = doc.createElement("template");
      template.innerHTML = getMetaTags(input);
      head.appendChild(template.content.cloneNode(true));

```

> error: TS2339 [ERROR]: Property 'content' does not exist on type 'Element'.
      head.appendChild(template.content.cloneNode(true));

Why is this happening and is there a workaround?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template#javascript

Thank you!

**Comment by b-fuze** [comment]
That TypeScript error can be resolved like this:
```ts
import { ..., HTMLTemplateElement } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
// ...
const template = doc.createElement("template") as HTMLTemplateElement;
```
as for the Javascript error that occurs when you run `.content.cloneNode(...)` I just pushed a fix for it in bbb4f93

**Comment by timonson** [comment]
Thank you!

Importing from master seems to work, but importing the module from the url with the commit throws this:

```ts
export {
  DOMParser,
  Element,
  type HTMLDocument,
  HTMLTemplateElement,
} from "https://raw.githubusercontent.com/b-fuze/deno-dom/bbb4f93727e22d7686076c0887aba920d9085e7e/deno-dom-wasm.ts";
```

> error: Module not found "https://raw.githubusercontent.com/b-fuze/deno-dom/bbb4f93727e22d7686076c0887aba920d9085e7e/src/dom/selectors/sizzle.js".
    at https://raw.githubusercontent.com/b-fuze/deno-dom/bbb4f93727e22d7686076c0887aba920d9085e7e/src/dom/selectors/sizzle-types.ts:1:20

**Comment by b-fuze** [comment]
@timonson it's in the latest published version v0.1.40
```ts
import {
  DOMParser,
  Element,
  type HTMLDocument,
  HTMLTemplateElement,
} from "https://deno.land/x/deno_dom@v0.1.40/deno-dom-wasm.ts";

const doc = new DOMParser().parseFromString("<template>foo bar</template>", "text/html")!;
const frag = doc.querySelector("template")! as HTMLTemplateElement;
console.log(frag.content.cloneNode().nodeName);
```
you don't need to use Github

**Comment by b-fuze** [comment]
Closing this as the relevant bug in Deno DOM was fixed in v0.1.40
